devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -118,14 +118,25 @@ private Map parseCatchAllMessage(String description) {
 	if (shouldProcessMessage(cluster)) {
 		switch(cluster.clusterId) {
 			case 0x0001:
-				resultMap = getBatteryResult(cluster.data.last())
+				if (cluster.command == 0x007) {
```

Comment:
I'll change to 8-bit `0x07`. In the original DTH, `//bind msg` was the comment used in `shouldProcessMessage`, but I'll change it to the way you suggested.